websites/T/Tempest Fansub/presence.ts

```diff
@@ -9,131 +9,119 @@ presence.on("UpdateData", async () => {
 			largeImageKey: "https://i.imgur.com/ppTaAAl.png",
 		},
 		searchParams = new URLSearchParams(location.search);
-	//Dino
-	if (pathname === "/dinozor-oyunu/") presenceData.details = "Dino oynuyor.";
-	//arama
-	else if (searchParams.get("s")) {
-		if (hostname === "tempestfansub.com")
-			presenceData.details = "Bir anime arıyor.";
-		if (hostname === "manga.tempestfansub.com")
+	if (hostname === "tempestscans.com") {
+		//Manga----------------------------------------------------------------------------------------------------------
```

Comment:
This is the correct declaration of a comment, please update them across all comments within code
```suggestion
		// Manga
```

Reply:
you are annoying me bro just accept